### DISCUSSION TITLE
en_core_web_lg vs en_core_web_trf

### DISCUSSION THREAD

**Discussion by hyperknot** [explosion/spaCy#12849] [Help: Model Advice] [answered]
Hi,

Is there any advantage for using the trf model, other than the 1-2% improvements listed on the model release?

I don't want to train anything myself, I'd just like to run the default pipeline on English text.

Also, what's not clear to me from the model releases: do trf models only work with a GPU?

+1 question: if I'm running en_core_web_lg, is there any point in running xx_sent_ud_sm for senter, or I should just enabled senter in the default en_core_web_lg pipeline? (From what I saw it's not enabled by default, right?)

**Comment by rmitsch** [comment, marked as answer]
Hi @hyperknot,

> Is there any advantage for using the trf model, other than the 1-2% improvements listed on the model release?

Well, how much exactly transformer models are better depends on your use case and data. In general no, their benefit is being more accurate, their drawback is being slower and needing more resources.

> Also, what's not clear to me from the model releases: do trf models only work with a GPU?

No, but they are _very_ slow on CPUs.

> +1 question: if I'm running en_core_web_lg, is there any point in running xx_sent_ud_sm for senter, or I should just enabled senter in the default en_core_web_lg pipeline? (From what I saw it's not enabled by default, right?)

The parser component also determines sentence boundaries. You don't need a separate senter component for that.
